org.go

```diff
@@ -2,15 +2,16 @@ package main
 
 //model aligned with v2-org-transformer
 type org struct {
-	UUID        string       `json:"uuid"`
-	ProperName  string       `json:"properName"`
-	Type        string       `json:"type"`
-	Identifiers []identifier `json:"identifiers,omitempty"`
+	UUID                   string                 `json:"uuid"`
+	ProperName             string                 `json:"properName"`
+	PrefLabel              string                 `json:"prefLabel"`
+	Type                   string                 `json:"type"`
+	AlternativeIdentifiers alternativeIdentifiers `json:"alternativeIdentifiers,omitempty"`
 }
 
-type identifier struct {
-	Authority       string `json:"authority"`
-	IdentifierValue string `json:"identifierValue"`
+type alternativeIdentifiers struct {
+	TME   []string `json:"TME,omitempty"`
+	Uuids []string `json:"uuids,omitempty"`
```

Comment:
This is actually aligned with the other transformers, e.g. https://github.com/Financial-Times/topics-transformer/blob/master/topic.go#L12 or https://github.com/Financial-Times/locations-transformer/blob/master/location.go#L12;  so I'd like to keep this way for now